src/ui/components/SecondaryToolbox/index.js

```diff
@@ -88,17 +88,19 @@ function InspectorPanel() {
   );
 }
 
-function SecondaryToolbox({ selectedPanel, setSelectedPanel, narrowMode }) {
+function SecondaryToolbox({ selectedPanel, setSelectedPanel, narrowMode, recordingTarget }) {
   const {
     userSettings: { show_react },
   } = hooks.useGetUserSettings();
+  const isNode = recordingTarget == "node";
   return (
-    <div className="secondary-toolbox">
+    <div className={`secondary-toolbox ${isNode && "node"}`}>
```

Comment:
I typically use`classnames` when I need to conditionally applying class names and thoroughly enjoy it: https://www.npmjs.com/package/classnames

Just a suggestion!

Reply:
yep - we use classnames too, just moving quickly